### PR TITLE
Add Coverage Notes article

### PR DESCRIPTION
**Pull request by J-Byron**

### CODE REVIEW

File: docs/work/coverage-notes/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+ 
+ # Introduction
+ When done correctly, coverage notes allow you to take the time you need off without resulting in any headaches for you or the team. In your coverage notes you should provide 3 things: What you are currently working on, what the status of your work is, and any other important notes that might be helpful to someone picking up the task. In conjunction with coverage notes, you should also talk to you PM(s) about anything else you might have missed. Your coverage notes should then be sent to your PM or most senior Dev.
+
+# Template
+
+So long as you provide notes for the 3 peices above, you are free to format the notes however you'd like. Below is an example ...
```
Comment (dholloran-ample) [inline docs/work/coverage-notes/README.md:7]: **peices** should be **pieces**

File: docs/work/coverage-notes/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+ 
+ # Introduction
+ When done correctly, coverage notes allow you to take the time you need off without resulting in any headaches for you or the team. In your coverage notes you should provide 3 things: What you are currently working on, what the status of your work is, and any other important notes that might be helpful to someone picking up the task. In conjunction with coverage notes, you should also talk to you PM(s) about anything else you might have missed. Your coverage notes should then be sent to your PM or most senior Dev.
```
Comment (dholloran-ample) [inline docs/work/coverage-notes/README.md:3]: 1. Grammarly is saying **coverage notes you should provide** should be **coverage notes, you should provide**
1. **you PM(s)** should be **your PM(s)**

File: docs/work/coverage-notes/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+# Template
+
+So long as you provide notes for the 3 peices above, you are free to format the notes however you'd like. Below is an example ... 
+
+## Example 
+
+## Client #1
+> Running Specs during build
+* Now that we have testing integrated into the netlify build, we need to get the broken tests to pass.
```
Comment (dholloran-ample) [inline docs/work/coverage-notes/README.md:13]: Nitpicky but could you capitalize Netlify?

File: docs/work/coverage-notes/README.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## Example 
+
+## Client #1
+> Running Specs during build
+* Now that we have testing integrated into the netlify build, we need to get the broken tests to pass. 
+*  I would say I am 70% done, I just need to refactor a bit more.
+> Style issues
+*  The staging site needs to be password protected again. Previously there was an issue that was causing the styling to get messed up, Sean has provided a possible solution in the task notes.
+*  Downloading the netlify CLI will allow you to test the password functionality locally
```
Comment (dholloran-ample) [inline docs/work/coverage-notes/README.md:17]: Could you capitalize Netlify here as well?

File: docs/work/coverage-notes/README.md (hunk 2 above)
Comment (dholloran-ample) [inline docs/work/coverage-notes/README.md:3]: @seancdavis What would be the correct terminology for the end of this line "most senior Dev"? Would it be "Dev Lead", "Dev Project Lead", "other developers on the project" or something else?

Also, would it be sent to the PM/developers or just the PM in the project Slack channel? That way it's visible to everyone and I think we have a channel for every project.

File: docs/work/coverage-notes/README.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+ # Introduction
+ When done correctly, coverage notes allow you to take the time you need off without resulting in any headaches for you or the team. In your coverage notes you should provide 3 things: What you are currently working on, what the status of your work is, and any other important notes that might be helpful to someone picking up the task. In conjunction with coverage notes, you should also talk to you PM(s) about anything else you might have missed. Your coverage notes should then be sent to your PM or most senior Dev.
+
+# Template
+
+So long as you provide notes for the 3 peices above, you are free to format the notes however you'd like. Below is an example ... 
+
+## Example 
+
+## Client #1
```
Comment (dholloran-ample) [inline docs/work/coverage-notes/README.md:11]: I think I said Client 1,2 when we talked about this but now that I'm reading it I'm not sure. How do you feel about it being "Client Name"? I'm a 2/5 on this so if you like it better the way it is I'm good with that.

**Review by dholloran-ample — CHANGES_REQUESTED** [review]
@J-Byron I left a few comments below. 

Also, a few tools to look into

1. I like to (sometimes forget to) run things like this through [Grammarly](https://www.grammarly.com/) it catches simple mistakes. 
1. [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) is good for when you're writing in VS Code (such as in the case of Markdown)

File: docs/work/coverage-notes/README.md (hunk 2 above)
Comment (seancdavis) [reply docs/work/coverage-notes/README.md:3]: My gut here is that we set the precedent to post in the #general channel in Slack. That way everyone has visibility and there's less ambiguity about who should receive notes. What do you think?

Comment (dholloran-ample) [reply docs/work/coverage-notes/README.md:3]: @seancdavis I think the #general channel on Slack is a good idea. Making it available to everyone was my goal with project-specific channels but #general gets everyone who needs it.

@J-Byron I think we could do something like "Your coverage notes should then be posted in the #general channel on Slack."

**Review by dholloran-ample — APPROVED** [review]

**Pull request merged**